server/server.go

```diff
@@ -120,7 +125,7 @@ func (cs *cribbageServer) ginPostCreateGame(c *gin.Context) {
 	}
 
 	// TODO investigate what it'll take to protobuf-ify our models
```

Comment:
🔪 remove this comment? I don't think it's helpful anymore

Reply:
FINALLY